jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -30,7 +30,7 @@ This release includes improvements in the following areas:
 * significant improvements in error messages, showing useful context
 * documentation generation logic and documentation, in particular `runnableExamples` now works in more contexts
 * JS, VM and nimscript backend are more consistent with C backend, allowing more modules to work with those backends, including the imports from `std/prelude`; the test suite now standardizes on testing stdlib modules on each major backend (C, JS, VM).
-* support for Apple silicon/M1, 32-bit RISC-V, CROSSOS, improved support for NodeJS backend
+* support for Apple silicon/M1, 32-bit RISC-V, armv8l, CROSSOS, improved support for NodeJS backend
```

Comment:
why duplicate this in two places?

Reply:
this is a short summary of the most salient points; the rest of the text has more details including links to PRs in some cases as well as grouping in semantic sections